src/main/java/BuyStockUseCase/BuyStockPresenter.java

```diff
@@ -5,6 +5,9 @@
 import java.io.IOException;
 
 public class BuyStockPresenter {
+    /**
+     * Presenter for the buy stock GUI
```

Comment:
Maybe a small comment on what the presenter does. For example what it connects, like GUI to UseCaseInteractor. This is just to show clear architecture is follwed.